app/validators/action_kit_fields.rb

```diff
@@ -77,23 +81,32 @@ def validate(record)
     end
   end
 
-  private
-
   # Does the name match an existing ActionKit field name, else
   # does it have a valid prefix ( +action_+, or +user_+ ).
   def has_valid_form
+    # puts "--- testing #{@name}, got #{is_predefined_by_ak} || #{has_valid_prefix}"
     is_predefined_by_ak || has_valid_prefix
   end
 
+  def self.is_predefined_by_ak(name)
+    ACTIONKIT_FIELDS_WHITELIST.include?(name.to_s)
+  end
+
+  def self.has_valid_form(name)
+    new(name).has_valid_form
+  end
+
+  private
+
   def has_valid_characters
-    @name =~ VALID_CHARS_RE
+    (@name =~ VALID_CHARS_RE).present?
   end
 
   def is_predefined_by_ak
-    ACTIONKIT_FIELDS_WHITELIST.include?(@name)
+    ACTIONKIT_FIELDS_WHITELIST.include?(@name.to_s)
   end
 
   def has_valid_prefix
-    @name =~ VALID_PREFIX_RE
+    (@name =~ VALID_PREFIX_RE).present?
```

Comment:
I'm taking your other suggestions, but not this one - `!(@name !~ VALID_PREFIX_RE)` is super arcane to me, as I've never seen `!~` before and it never occurs in our codebase, whereas `=~` is very common. I also think that methods beginning in `is_` or `has_` should return booleans.